pytorch_lightning/accelerators/accelerator.py

```diff
@@ -75,7 +75,6 @@ def auto_device_count() -> int:
     def is_available() -> bool:
         """Detect if the hardware is available."""
 
-    @staticmethod
-    @abstractmethod
-    def name() -> str:
-        """Name of the Accelerator."""
+    @classmethod
+    def register_accelerators(cls, accelerator_registry: Dict) -> None:
```

Comment:
Typing is wrong. should be the actual accelerator registry

Reply:
Even if it subclasses Dict?